### DISCUSSION TITLE
[FEATURE] PromptPay

### DISCUSSION THREAD

**Discussion by korewaChino** [juspay/hyperswitch#2825] [Ideas / Feature Requests]
### Feature Description

Implement PromptPay support for Hyperswitch

PromptPay is a payment system in Thailand that allows users to wire funds using a Thai ID or a phone number, instead of bank details.

Vendors can also create invoices for specific amounts.

Read more here at the BoT: https://www.bot.or.th/en/financial-innovation/digital-finance/digital-payment/promptpay.html

### Possible Implementation

Implement an API shim for PromptPay

### Have you spent some time checking if this feature request has been raised before?

- [X] I checked and didn't find a similar issue

### Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/juspay/hyperswitch/blob/main/docs/CONTRIBUTING.md)

### Are you willing to submit a PR?

No, but I'm happy to collaborate on a PR with someone else

**Comment by NarsGNA** [comment]
This is an interesting use case, we will get back shortly
